db/fixtures/miq_alerts.yml

```diff
@@ -321,3 +321,45 @@
             value: prod
   :db: Vm
   :guid: 89db0be8-c240-11de-a3be-000c290de4f9
+- description: Middleware Heap Max > 90%
+  options:
+    :notifications:
+      :delay_next_evaluation: 600
+      :evm_event: {}
+  db: MiddlewareServer
+  responds_to_events: hawkular_alert
+  hash_expression:
+    :eval_method: mw_heap_used
+    :mode: internal
+    :options:
+      :value_mw_greater_than: '90'
+      :value_mw_less_than: '0'
+  guid: 017f8cbf-fb9b-4417-9b0a-c0fc02d8e0a2
+- description: Middleware Non Heap > 90%
+  options:
+    :notifications:
+      :delay_next_evaluation: 600
+      :evm_event: {}
+  db: MiddlewareServer
+  responds_to_events: hawkular_alert
+  hash_expression:
+    :eval_method: mw_non_heap_used
+    :mode: internal
+    :options:
+      :value_mw_greater_than: '90'
+      :value_mw_less_than: '0'
+  guid: b6fa8fb2-97fd-4c95-8374-bf374fc7ab6f
+- description: Middleware GC > 10 sec/minute
+  options:
+    :notifications:
+      :evm_event: {}
+      :delay_next_evaluation: 600
+  db: MiddlewareServer
+  responds_to_events: hawkular_alert
+  hash_expression:
+    :eval_method: mw_accumulated_gc_duration
+    :mode: internal
+    :options:
+      :mw_operator: ">"
```

Comment:
I am wondering why here the > operator is used but in other alerts above there is "greater than" used? (I am not familiar with the area, there might be a good reason for that)

Reply:
That's because the mapping in our alert manager
See:
https://github.com/ManageIQ/manageiq-providers-hawkular/blob/388a28b2c07a45f0f319b247f8b38fea949db08f/app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb#L205-L212

Reply:
The alert title is ```Middleware GC > 10 sec/minute```. I don't see where the time threshold is specified.